### PR TITLE
Add size as variable to lang string

### PR DESCRIPTION
**Pull request by geichelberger**
Fixes: https://github.com/opencast/opencast-admin-interface/issues/535

### CODE REVIEW

File: src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
 		}
 	},
 	"CONFIRMATIONS": {
-		"EMBEDDING_CODE": "Embedding code has been copied to the clipboard:",
+		"EMBEDDING_CODE": "Embedding code has been copied to the clipboard: {{ size }}",
```
Comment (ziegenberg) [inline src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json:100]: Please only change `en_US` as Crowdin will overwrite the others.

**Review by ziegenberg — CHANGES_REQUESTED** [review]

File: src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json (hunk 1 above)
Comment (geichelberger) [reply src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json]: So, is there no way to bulk update from the code?

Comment (ziegenberg) [reply src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json]: See https://docs.opencast.org/develop/developer/#participate/localization/

> The English translation (en-US) acts as source language that is translated to other languages using Crowdin. While all translation files are located in the Opencast code repository, only the English translation should be modified in the code repository - all other translation files are downloaded from Crowdin.
> 
> **Important**: _All translation files for languages other than English (en-US) are downloaded from Crowdin. Modifications to the translation files in the Opencast code repository will be regularly overwritten and therefore will be lost!_

Comment (ziegenberg) [reply src/i18n/org/opencastproject/adminui/languages/lang-da_DK.json]: Das gilt auch für Editor und AmdinUI

**Pull request merged**